### PR TITLE
Improve picture fades

### LINKED ISSUE #510: Allow fade skipping
Currently when the eidos logo fades up, you have to wait for the fade to finish before you can skip it with ESC or ACTION. It should allow to skip at any time though.

I suppose this affects all instances of fades?

### PR DESCRIPTION
**Pull request by rr-**
#### Checklist

- [x] I have read the [coding conventions](https://github.com/rr-/Tomb1Main/blob/master/CONTRIBUTING.md#coding-conventions)
- [x] I have added a changelog entry about what my pull request accomplishes

#### Descripton

Resolves #510

### CODE REVIEW

**Comment by walkawayy** [comment]
Works great! Yay for finally being able to skip through the opening Eidos picture faster haha.

**Review by walkawayy — APPROVED** [review]

File: src/game/gameflow.c [hunk 1]
```diff
@@ -1173,21 +1174,32 @@ GameFlow_InterpretSequence(int32_t level_num, GAMEFLOW_LEVEL_TYPE level_type)
...
+                if (!g_InputDB.any) {
+                    Output_InitialisePolyList();
+                    Output_CopyPictureToScreen();
+                    Output_DumpScreen();
+                    Shell_Wait(data->display_time);
+                }
 
                 // fade out
                 Output_FadeToBlack(true);
                 while (Output_FadeIsAnimating()) {
```
Comment (walkawayy) [inline src/game/gameflow.c:1193]: Gonna be annoying and say maybe make all three input checks consistent? (unless there's a reason they are structured differently).

`while (Output_FadeIsAnimating() && !g_InputDB.any)`?

**Review by walkawayy — CHANGES_REQUESTED** [review]

File: src/game/gameflow.c (hunk 1 above)
Comment (rr-) [reply src/game/gameflow.c]: Simplified

**Review by walkawayy — APPROVED** [review]

**Review by walkawayy — CHANGES_REQUESTED** [review]
Wait I checked credits and somehow they are broken now. It skips all the credit images straight to the final stats screen.

**Comment by rr-** [comment]
.

**Review by walkawayy — APPROVED** [review]
Works well!

**Comment by rr-** [comment]
Resolved conflicts; merging

**Pull request merged**